src/App.js

```diff
@@ -1,47 +1,52 @@
 import React from 'react';
-import {
-  BrowserRouter as Router, Routes, Route, Link,
-} from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { verifyCredentials } from './redux/login/loginAction';
+import HomePage from './components/homepage/HomePage';
+import LoginForm from './components/login/loginForm';
+import Detail from './components/Detail';
+import DeletePage from './components/delete/deletePage';
+import Reservations from './components/Reservations/reservations';
+import MyReservations from './components/Reservations/myReservations';
+import AddNewCar from './components/forms/AddNewCar';
+import './App.css';
+import Navbar from './components/Navbar/navbar';
 
 function App() {
+  const authDetails = useSelector((state) => state.loginPage);
+  const dispatch = useDispatch();
+
+  const submitCredentials = (unameVal) => {
+    dispatch(verifyCredentials(unameVal));
+  };
+
   return (
     <Router>
-      <div>
-        <nav>
-          <ul>
-            <li>
-              <Link to="/">Home</Link>
-            </li>
-            <li>
-              <Link to="/about">About</Link>
-            </li>
-            <li>
-              <Link to="/users">Users</Link>
-            </li>
-          </ul>
-        </nav>
-
-        {/* A <Switch> looks through its children <Route>s and
-        renders the first one that matches the current URL. */}
+      <main style={{ display: 'flex' }}>
```

Comment:
- [ ] Avoid using the `style` attribute here, using CSS with classes is a better practice.